frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -314,14 +335,16 @@ export function DashCardVisualization({
         downloadsEnabled={downloadsEnabled}
         question={question}
         result={mainSeries}
-        dashcardId={dashcard.id}
         dashboardId={dashboard.id}
+        dashcardId={dashcard.id}
         token={
           isJWT(dashcard.dashboard_id)
             ? String(dashcard.dashboard_id)
             : undefined
         }
         uuid={isUuid(dashcard.dashboard_id) ? dashcard.dashboard_id : undefined}
+        onEdit={editCard}
+        onEditLabel={t`Edit visualization`}
```

Comment:
Since we will now have two different "editing" behaviors, I think we should make it explicit which props refer to each behavior. I'd suggest we rename `onEdit` to something like `onEditVisualization`